tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -1103,6 +1108,8 @@ public bool DiagnosticSourceEnabled
 
         internal ImmutableDynamicSettings DynamicSettings { get; init; } = new();
 
+        internal string? JsonConfigurationFilePath { get; }
```

Comment:
Technically we can have any number of json file paths
```suggestion
        internal List<string> JsonConfigurationFilePaths { get; } = new();
```